### PR TITLE
Add pytest package

### PR DESCRIPTION
**Pull request by jaguarfi**
# Description

Add pytest package to the **grade-python** image. 

# Testing

I created the docker image locally and I tested it with a modified copy of the [hoscic exercises](https://version.aalto.fi/gitlab/course/hoscicomp-exercises/-/tree/2020-summer)

# Have you updated the README or other relevant documentation?

The README file was updated

# Is it [Done](https://wiki.aalto.fi/display/EDIT/Definition+of+Done)?

*Clean up your git commit history before submitting the pull request!*

- [ ] I (developer) have created unit tests and the tests pass
- [ ] I (developer) have created functional tests (Selenium tests) if applicable
- [X] I (developer) have tested the changes manually
- [ ] Reviewer has finished the code review
- [ ] After the review, the developer has made changes accordingly
- [ ] Customer/Teacher has accepted the implementation of the feature

### CODE REVIEW

**Comment by raphendyr** [comment]
I think pytest is so generic, it should be part of the base python image.

**Comment by jaguarfi** [comment]
> I think pytest is so generic, it should be part of the base python image.
@raphendyr it sounds like a good idea. Maybe @Mankro want to share his opinion about this matter.

**Comment by raphendyr** [comment]
Can you comment how much the new layer takes disk space? Like if it's kilobytes, then it could trivialyl be part of the base layer.

**Comment by jaguarfi** [comment]
Do you mean this size?. If so, it only adds 3MB
image

**Comment by raphendyr** [comment]
So, around 3MB, I guess that is fine from my point of view. Other tags are way bigger (`dev` images are my local builds):

```
apluslms/grade-python      y2-dev                 8d18d0316cd7        2 months ago        432MB
apluslms/grade-python      xls-dev                b229cfa93d61        2 months ago        228MB
apluslms/grade-python      rdf-dev                2d31335522a0        2 months ago        230MB
apluslms/grade-python      math-dev               0d6a79dfa0a9        2 months ago        356MB
apluslms/grade-python      dev                    73589b94872f        2 months ago        226MB
apluslms/grading-base      dev                    0f156a4bd2eb        2 months ago        187MB
```

**Comment by Mankro** [comment]
Does the pytest package conflict with any other packages installed in the container? Probably not, so let's include it in the base grade-python container then. @jaguarfi the content of the PR becomes then quite different. You will need to modify the main `Dockerfile`. If pytest is available as a Debian package, then Jaakko has preferred those over using pip install. If not, then just use `pip install` or actually the container has its own version `pip_install`.

**Comment by jaguarfi** [comment]
I search online and it seems that there is no conflict between **pytest** and the other packages installed in this docker image.

I change the title and content of the PR, now, It should be ready for review.

File: Dockerfile [hunk 1]
```diff
@@ -9,6 +9,7 @@ RUN apt_install \
     python3 \
     python3-pip \
     python3-setuptools \
+    python3-pytest \
```
Comment (raphendyr) [inline Dockerfile:12]: You could sort these lines.. MIght be easier to read after a while.

**Comment by jaguarfi** [comment]
I forced the push the changes since the requested change is easy to track. I guess the PR is ready now :)

**Review by Mankro — APPROVED** [review]
Thanks!

**Comment by Mankro** [comment]
The new container with this update will be named `apluslms/grade-python:3.7-3.3-3.1u1` once it has been built in Docker Hub.

**Pull request merged**